src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -128,13 +128,21 @@ namespace InternalHeatGains {
 
     // Data
     // MODULE PARAMETER DEFINITIONS:
-    int constexpr ITEClassNone(0);
-    int constexpr ITEClassA1(1);
-    int constexpr ITEClassA2(2);
-    int constexpr ITEClassA3(3);
-    int constexpr ITEClassA4(4);
-    int constexpr ITEClassB(5);
-    int constexpr ITEClassC(6);
+    enum class ITEClass
+    {
+        Invalid = -1,
+        None, // (0)
```

Comment:
Is `None` really a legitimate choice? Or is it the same as `Invalid`?

Reply:
Yes, it is a legitimate and the default values to go. `None` basically means "no restriction" from the guideline. It is is different from "Invalid".

Reply:
Got it, thx!